### DISCUSSION TITLE
what param for   "consumer.override.max.poll.records" and "consumer.override.max.partition.fetch.bytes" should be used for Confluecnt Cloud

### DISCUSSION THREAD

**Discussion by zcistkidd** [ClickHouse/clickhouse-kafka-connect#167] [Q&A]
In my current config for Confluent Cloud BYOC :, I have the following 2 params set at
```
  "consumer.override.max.poll.records": "1000",
  "consumer.override.max.partition.fetch.bytes": "10240000",
```
And it kept timeout with following WARN msg  at polling data:
**consumer poll timeout has expired. This means the time between subsequent calls to poll() was longer than the configured max.poll.interval.ms, which typically implies that the poll loop is spending too much time processing messages. You can address this either by increasing max.poll.interval.ms or by reducing the maximum size of batches returned in poll() with max.poll.records.**
And here is my setup:
[clickhouse-kafka-connect] 0.18 and using standard Confluent Cloud Plan,

**Comment by zcistkidd** [comment]
Also tried with same topic and 1000 batch size, HTTP sink to Clickhouse is working as expected, but we do wanna try this connector as EOS is what we need

**Comment by mshustov** [comment]
@zcistkidd There might be a few root causes: message processing and a network issue.

Since you don't control the network when running in `Confluent Cloud BYOC`, let's focus on the potential message processing slowness.
What is an average message size? Can we have a message example (obfuscated if necessary) to test message processing logic? Could you also provide the destination table schema?
